tests/formats/sync_protocol/light_client_sync.md

```diff
@@ -0,0 +1,59 @@
+# Light client sync tests
+
+This series of tests provides reference test vectors for validating that a light client implementing the sync protocol can sync to the latest block header.
+
+## Test case format
+
+### `meta.yaml`
+
+```yaml
+genesis_validators_root: Bytes32  -- string, hex encoded, with 0x prefix
+trusted_block_root: Bytes32       -- string, hex encoded, with 0x prefix
+```
+
+### `bootstrap.ssz_snappy`
+
+An SSZ-snappy encoded `bootstrap` object of type `LightClientBootstrap` to initialize a local `store` object of type `LightClientStore` using `initialize_light_client_store(trusted_block_rooot, bootstrap)`.
+
+### `steps.yaml`
+
+The steps to execute in sequence. There may be multiple items of the following types:
+
+#### `process_slot` execution step
+
+The function `process_slot_for_light_client_store(store, current_slot)`
+should be executed with the specified parameters:
+
+```yaml
+{
+    current_slot: int  -- integer, decimal
+}
+```
+
+After this step, the `store` object may have been updated.
+
+#### `process_update` execution step
+
+The function `process_light_client_update(store, update, current_slot, genesis_validators_root)` should be executed with the specified parameters:
+
+```yaml
+{
+    update: string             -- name of the `*.ssz_snappy` file to load
```

Comment:
suggest adding a note about the suffix rules

Reply:
So far, the suffixes are just an implementation detail to avoid name clashes, the test runner does not need to know about their meaning, i.e., they can just take the filename from here and run the test without further processing the name.